nemo/collections/asr/modules/rnnt.py

```diff
@@ -50,6 +50,525 @@
 from nemo.utils import logging
 
 
+class StatelessNet(torch.nn.Module):
```

Comment:
Move this class out of this file somewhere else. This file is for final level modules of RNNT not intermediate modules.

Reply:
Done.